substrate/client/rpc-api/src/author/mod.rs

```diff
@@ -41,6 +55,15 @@ pub trait AuthorApi<Hash, BlockHash> {
 	#[method(name = "author_rotateKeys")]
 	fn rotate_keys(&self) -> Result<Bytes, Error>;
 
+	/// Generate new session keys and returns the corresponding public keys.
+	///
+	/// The `owner` should be something that can be used on chain for verifying the ownership of the
+	/// generated keys using the returned `proof`. For example `owner` could be set to the account
+	/// id of the account registering the returned public session keys. The actual data to pass for
+	/// `owner` depends on the runtime logic verifying the `proof`.
```

Comment:
Maybe we can add an example here?

Something like: "For example, "session pallet" `set_keys` requires `owner` to be the account used to sign the extrinsic."